src/control/cmd/daos_server/auto_test.go

```diff
@@ -260,6 +258,30 @@ func TestDaosServer_Auto_confGen(t *testing.T) {
 			WithTargetCount(18).WithHelperStreamCount(4),
 	}
 
+	var defCoresPerNuma uint32 = 26
+	var defNumaCount uint32 = 2
+	defMemInfo := common.MemInfo{HugepageSizeKiB: 2048}
```

Comment:
true for x86_64, but we should be agnostic to HP size and/or query it, rather than assume it. e.g. aarch64 is different..

Reply:
From my understanding, this is only for unit testing purpose.